app/src/main/java/com/sun/chat_04/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,137 @@
+package com.sun.chat_04.ui.login
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.v4.content.ContextCompat
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.Toast
+import com.facebook.CallbackManager
+import com.facebook.FacebookCallback
+import com.facebook.FacebookException
+import com.facebook.login.LoginResult
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+import com.sun.chat_04.R
+import com.sun.chat_04.R.id
+import com.sun.chat_04.R.layout
+import com.sun.chat_04.util.Constants
+import kotlinx.android.synthetic.main.activity_login.btnLoginEmailPass
+import kotlinx.android.synthetic.main.activity_login.btnLoginFace
+import kotlinx.android.synthetic.main.activity_login.btnLoginFb
+import kotlinx.android.synthetic.main.activity_login.edtEmailLogin
+import kotlinx.android.synthetic.main.activity_login.edtPassLogin
+import kotlinx.android.synthetic.main.activity_login.pgbSignIn
+
+class LoginActivity : AppCompatActivity(), View.OnClickListener
+    , LoginContract.View, TextWatcher {
+
+    private lateinit var presenter: LoginPresenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(layout.activity_login)
+        initComponents()
+        initLoginFacebook()
+    }
+
+    private fun checkButton() {
+        if (isValidEmailAndPassword()) {
+            btnLoginEmailPass.setBackgroundColor(
+                ContextCompat.getColor(
+                    this@LoginActivity,
+                    R.color.colorPinkDark
+                )
+            )
+            btnLoginEmailPass.isClickable = true
```

Comment:
check lại convention
```suggestion
            buttonLoginViaEmailPassword.isClickable = true
```